### DISCUSSION TITLE
Reopening Discussion: Update to .NET 7

### DISCUSSION THREAD

**Discussion by chavi362** [ppy/osu#26348] [Ideas]
Background:

Almost two years ago, there was a discussion about updating the osu! game client to .NET 7 (#21742), where the decision was made to delay the update due to other priorities.
Request for Reconsideration:

I would like to reopen the discussion regarding the transition to .NET 7. The .NET 7 framework has seen significant advancements since our last discussion, and more developers and users are embracing this version.
Recent Progress:

In the past two years, the osu! project has evolved, and the .NET community has made substantial improvements. I believe that updating to .NET 7 is now more relevant and could bring numerous benefits to the osu! game client.
Proposal:

I am willing to take on the task of transitioning the osu! game client to .NET 7. I have reviewed the latest .NET 7 documentation, and I am confident that this update can be beneficial for the project.
Reasons to Update:

Proposed Change:
I propose transitioning the project to utilize .NET 7 for the following reasons:

Advancements and Improvements: .NET 7 introduces new features, improvements, and optimizations that can benefit the osu! game client.

Long-Term Support: .NET 7 is designed for long-term support, ensuring stability and security for the project.

Community Adoption: With more developers and users shifting to .NET 7, updating the osu! game client will align it with the latest industry standards.

 Long-Term Support: .NET 7 is designed for long-term support, providing stability and security for the project.

Request for Collaboration:

I would appreciate feedback from the maintainers and the community on reconsidering the update to .NET 7.  I am prepared to work on the transition.
Please give me permission to work on this
Thanks
Chavi362

**Comment by peppy** [comment]
No thanks. We'll hand this ourselves when we're ready.
